package.json

```diff
@@ -40,7 +40,7 @@
     "@fengweichong/react-native-gzip": "github:cozy/react-native-gzip#1.2.0",
     "@mythologi/react-native-receive-sharing-intent": "2.2.1",
     "@notifee/react-native": "^7.8.0",
-    "@op-engineering/op-sqlite": "^6.2.11",
+    "@op-engineering/op-sqlite": "^14.0.1",
```

Comment:
No Breaking change? :o

Reply:
You are right I removed it 👍 Thank you for the investigation.